modules/ztp-filtering-ai-crs-using-siteconfig.adoc

```diff
@@ -0,0 +1,89 @@
+
+Using filters, you can easily customize `SiteConfig` custom resources (CRs) to include or exclude other CRs for use in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline.
+
+The following example configures a custom single-node cluster `SiteConfig` CR that excludes the `siteconfig/extra-manifest/03-sctp-machine-config-worker.yaml` file from the installation.
+
+.Prerequisites
+
+. You configured the hub cluster for generating the required installation and policy CRs.
+
+. You created a Git repository where you manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+.Procedure
+
+. Create a set of custom manifest CRs that the ZTP pipeline uses to customize the cluster installs. The following example illustrates the directory structure:
++
+[source,text]
+----
+.
+├── siteconfig
+│     ├── site1-sno-du.yaml
+│     └── user-custom-manifest
+│           └── custom-sctp-machine-config-worker.yaml
+└── extra-manifest
+      └── 03-sctp-machine-config-worker.yaml
+----
+
+. Edit the custom `site1-sno-du.yaml` file to set the `exclude` files, for example:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "site1-sno-du"
+  namespace: "site1-sno-du"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-{product-version}"
+  sshPublicKey: "<ssh_public_key>" <1>
+  clusters:
+  - clusterName: "site1-sno-du"
+    extraManifestPath: "<custom_manifest_folder>" <2>
+    extraManifests: <3>
+      filter:
+        exclude:
+          - 03-sctp-machine-config-worker.yaml
+  ...
+----
+<1> Replace `<ssh_public_key>` with the SSH public key that you use to access the cluster.
+<2> Replace `<custom_manifest_folder>` with the name of the `extraManifestPath` folder that contains the custom install CRs, for example `user-custom-manifest/`.
+<3> Set the required `include` or `exclude` statements for the install. You can include or exclude any CRs from the folder specified in the `extraManifestPath` field.
+
+. Save the `SiteConfig` CRs  and `/user-custom-manifest` CRs and push them to the site configuration repository.
++
+The ZTP pipeline monitors the CRs in the `/user-custom-manifest` directory and adjusts what CRs it applies based on the `SiteConfig` filter instructions.
+
+. Optional: To remove all CRs at install time including default RAN and user defined CRs, apply the following YAML in the `SiteConfig` CR:
++
+[source,yaml]
+----
+- clusterName: "site1-sno-du"
+  extraManifests:
+    filter:
+      inclusionDefault: exclude
+----
+
+. Optional: To exclude all extra manifests except for a set of custom CRs, set `inclusionDefault` to `exclude`, add the custom `extraManifestPath` folder location, and specify the `include` files, for example:
```

Comment:
> Generally, when you specify the name of a command, parameter, or option in text, specify the word command, parameter, or option after the name.
Command, parameter, and option names are not translated; without the identifying noun, users might be confused about what the name represents.

"directory" is used much more often in the collection than "folder" is.